### DISCUSSION TITLE
Integrate Chat Copilot

### DISCUSSION THREAD

**Discussion by damthieuquang** [intitni/CopilotForXcode#298] [Ideas]
Hi @intitni, as you know, copilot has a new feature chat with copilot. Do you have any plans to integrate into CopilotForXcode?

**Comment by intitni** [comment]
It's not possible at the moment, GitHub Copilot Chat doesn't have any public API.

**Comment by wimvandenborre** [comment]
Do you know if it's already possible? Can we setup a token for it?

**Reply by intitni** [reply]
It's not possible yet.
